### PR TITLE
ADMIN-322 fix: change color coding of equipment defaulters card values to bette…

### PR DESCRIPTION
**Pull request by ahadziii**
…r display the meaning

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Screenshots/Videos (if appropriate):

Screenshot 2022-10-15 at 9 14 55 PM

Screenshot 2022-10-15 at 9 16 53 PM

### CODE REVIEW

File: web-react-ts/src/pages/campaigns/components/buttons/DefaulterDetailsCard.tsx [hunk 1]
```diff
@@ -56,27 +56,43 @@ function DefaulterDetailsCard(props: DefaulterDetailsCardProps) {
...
+          }
+        >
           Fellowship Not Filled Count:{' '}
           {church?.fellowshipEquipmentNotFilledCount}
         </div>
         {props.route === 'constituency/fellowship' ? null : (
-          <div className="text-danger">
+          <div
+            className={
+              church?.constituencyEquipmentNotFilledCount === 0 ? '' : 'bad'
```
Comment (jaedag) [inline web-react-ts/src/pages/campaigns/components/buttons/DefaulterDetailsCard.tsx:93]: This being zero should be good rather.

File: web-react-ts/src/pages/campaigns/components/buttons/DefaulterDetailsCard.tsx [hunk 2]
```diff
@@ -56,27 +56,43 @@ function DefaulterDetailsCard(props: DefaulterDetailsCardProps) {
...
+          >
             Constituency Filled Count:{' '}
             {church?.constituencyEquipmentFilledCount}
           </div>
         )}
 
-        <div className="text-danger">
+        <div
+          className={
+            church?.fellowshipEquipmentNotFilledCount === 0 ? '' : 'bad'
```
Comment (jaedag) [inline web-react-ts/src/pages/campaigns/components/buttons/DefaulterDetailsCard.tsx:84]: this being zero should also  be good

File: web-react-ts/src/pages/campaigns/components/buttons/DefaulterDetailsCard.tsx [hunk 3]
```diff
@@ -56,27 +56,43 @@ function DefaulterDetailsCard(props: DefaulterDetailsCardProps) {
...
+            className={church.equipmentRecord.pulpits === null ? 'bad' : 'good'}
+          >
             Constituency Filled:{' '}
             {church.equipmentRecord.pulpits === null ? 'No' : 'Yes'}
           </div>
         ) : (
-          <div className="text-success">
+          <div
+            className={
+              church?.constituencyEquipmentFilledCount === 0 ? '' : 'good'
```
Comment (jaedag) [inline web-react-ts/src/pages/campaigns/components/buttons/DefaulterDetailsCard.tsx:74]: This condition should be checking whether constituencies fill is equal to the Total number of constituencies

File: web-react-ts/src/pages/campaigns/components/buttons/DefaulterDetailsCard.tsx [hunk 4]
```diff
@@ -56,27 +56,43 @@ function DefaulterDetailsCard(props: DefaulterDetailsCardProps) {
           </div>
         )}
 
-        <div className="text-success">
+        <div
+          className={church?.fellowshipEquipmentFilledCount === 0 ? '' : 'good'}
```
Comment (jaedag) [inline web-react-ts/src/pages/campaigns/components/buttons/DefaulterDetailsCard.tsx:60]: This one should also be checking whether this number is equal to the total number of fellowships

**Review by jaedag — CHANGES_REQUESTED** [review]

File: web-react-ts/src/pages/campaigns/components/buttons/DefaulterDetailsCard.tsx (hunk 1 above)
Comment (ahadziii) [reply web-react-ts/src/pages/campaigns/components/buttons/DefaulterDetailsCard.tsx:93]: tricky one here. we also want that when no forms have been filled, we should see white

**Review by jaedag — APPROVED** [review]

**Pull request merged**